core/src/main/java/org/elasticsearch/discovery/DiscoveryStats.java

```diff
@@ -64,4 +80,5 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
     public PendingClusterStateStats getQueueStats() {
         return queueStats;
     }
+    public PublishClusterStateStats getPublishStats() { return publishStats; }
```

Comment:
Would you add a newline before this method, and wrap the method definition over multiple lines please?